src/bosh_aws_cpi/lib/cloud/aws/resource_wait.rb

```diff
@@ -77,15 +77,18 @@ def self.for_volume_modification(args)
       target_state = args.fetch(:state) { raise ArgumentError, 'state symbol required' }
       valid_states = ['modifying', 'optimizing', 'completed', 'failed']
       validate_states(valid_states, target_state)
+      wait_time_factor = args.fetch(:wait_time_factor, 1).to_i
```

Comment:
`wait_time_factor` is a required option with a default. I'm not sure why we need another default here.

Reply:
For multi cpi the properties are merged from context: https://github.com/cloudfoundry/bosh-aws-cpi-release/blob/master/src/bosh_aws_cpi/bin/aws_cpi#L22

Which means they bypass the default set in the erb template.

Reply:
Plus, having a default here is handy so that the argument can be omitted in most cases.
This `for_volume_modification()` method could be called from other places in the CPI code.
Let's keep it like this.